### ISSUE TITLE
Details of accessibilitysummary violation is still wrong

### ISSUE THREAD

**Issue by Changilainen** [daisy/ace-gui#185]
When ebook is missing accessibilitysummary field, the report is still saying it's a must to have. See the attachment where it says "**Publications must declare the 'schema:accessibilitySummary' metadata**". Shouldn't the word 'must' be replaced with the word 'should'?

![Image](https://github.com/user-attachments/assets/eccd1707-ccbe-4b8a-9327-62b143b03442)

**Comment by danielweck** [comment]
ah yes, the severity was downgraded ('moderate' in your screenshot) but all the language translations must be updated to reflect this!
